homeassistant/helpers/schema_config_entry_flow.py

```diff
@@ -31,9 +31,8 @@ class SchemaFlowFormStep(SchemaFlowStep):
     """Define a config or options flow form step."""
 
     schema: vol.Schema | Callable[
-        [SchemaConfigFlowHandler | SchemaOptionsFlowHandler, dict[str, Any]],
-        vol.Schema | None,
-    ] | None
+        [SchemaCommonFlowHandler], vol.Schema | None
+    ] | None = None
```

Comment:
I think the docstring needs to be updated, because this is no longer true?
> the function will be passed the handler, which is
    either an instance of SchemaConfigFlowHandler or SchemaOptionsFlowHandler, *and the
    union of config entry options and user input from previous steps.*